test/fixtures/e2e/amp-list/amp-list.ssr.html

```diff
@@ -0,0 +1,29 @@
+<!doctype html>
+<html ⚡ allow-viewer-render-template>
```

Comment:
`⚡4email` right?

Reply:
right. Though for testing ssr, allow-viewer-render-template is enough along with the capability of viewerRenderTemplate in the viewer.

Reply:
when is the `⚡4email` needed? just for triggering the right validation rules, or does it change the behavior of the runtime?

Reply:
⚡4email is only for validation. It does not change the behavior of runtime at all though the sanitizer used by the amp-mustache extension will soon have context of the html format being used in order to sanitize templates accordingly.

Reply:
This document uses the amp4email CSS boilerplate so we should at least be consistent.